modules/sfp_opencorporates.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_opencorporates
+# Purpose:      SpiderFoot plug-in for retrieving company location and previous
+#               company names from OpenCorporates.
+#
+# Author:      Brendan Coles <bcoles@gmail.com>
+#
+# Created:     2018-10-21
+# Copyright:   (c) Brendan Coles 2018
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import re
+import urllib
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+class sfp_opencorporates(SpiderFootPlugin):
+    """OpenCorporates:Passive,Footprint,Investigate:Search Engines::Look up company information from OpenCorporates."""
+
+    # Default options
+    opts = {
+        'confidence': 100
+    }
+
+    # Option descriptions
+    optdescs = {
+        'confidence': "Confidence that the search result objects are correct (numeric value between 0 and 100)."
```

Comment:
It's not a field that's used today, so not sure it's worth adding.

Reply:
What do you mean? This field is used.